src/test/java/com/vmware/gemfire/testcontainers/GemFireTestcontainersTest.java

```diff
@@ -184,15 +184,19 @@ public void testStartWithPdx() {
 
   @Test
   public void testWithSecurityManagerOnClasspath() {
+    String username = "cluster,data";
+    String password = "cluster,data";
+
     try (GemFireCluster cluster = new GemFireCluster()) {
-      cluster.withClasspath(ALL_GLOB, "build/classes/java/test", "out/test/classes")
+      cluster.withClasspath(ALL_GLOB, "out/test/classes", "build/classes/java/test")
           .withGemFireProperty(ALL_GLOB, "security-manager", SimpleSecurityManager.class.getName())
-          .withGemFireProperty(ALL_GLOB, "security-username", "cluster")
-          .withGemFireProperty(ALL_GLOB, "security-password", "cluster")
+          .withGemFireProperty(ALL_GLOB, "security-username", username)
+          .withGemFireProperty(ALL_GLOB, "security-password", password)
           .acceptLicense()
           .start();
 
-      cluster.gfsh(true, "list members",
+      cluster.gfsh(true, Credentials.of(username, password),
+          "list members",
           "create region --name=FOO --type=REPLICATE");
```

Comment:
I think a `withCredentials()` should be available on the container rather than thrown in on the `gfsh` command. Also, `withCredentials()` could replace the explicit setting of `security-username` and `security-password`.

Reply:
OK, so I reworked it quite a bit. There is now a small Builder that can be retrieved using `cluster.gfshBuilder()` where one can configure creds as well as TLS options. That then allows you to create a `Gfsh` instance that you can `run()` commands with.

I don't think adding `withCredentials` on the container is the right thing (at this point).

In general, I'd prefer configuration and setup to be fairly explicit. I think it's a fine line between making things 'easier' for a user and hiding too much of the details that they end up wondering (and possibly needing to debug) what's happening behind the curtain.